crates/gitbutler-tauri/src/menu.rs

```diff
@@ -303,17 +304,17 @@ pub fn handle_event<R: Runtime>(event: &WindowMenuEvent<R>) {
 
     'open_link: {
         let result = match event.menu_item_id() {
-            "help/documentation" => open::that("https://docs.gitbutler.com"),
-            "help/github" => open::that("https://github.com/gitbutlerapp/gitbutler"),
-            "help/release-notes" => {
-                open::that("https://discord.com/channels/1060193121130000425/1183737922785116161")
-            }
+            "help/documentation" => open::open_url("https://docs.gitbutler.com"),
+            "help/github" => open::open_url("https://github.com/gitbutlerapp/gitbutler"),
+            "help/release-notes" => open::open_url(
+                "https://discord.com/channels/1060193121130000425/1183737922785116161",
+            ),
             "help/report-issue" => {
-                open::that("https://github.com/gitbutlerapp/gitbutler/issues/new")
+                open::open_url("https://github.com/gitbutlerapp/gitbutler/issues/new")
             }
-            "help/discord" => open::that("https://discord.com/invite/MmFkmaJ42D"),
-            "help/youtube" => open::that("https://www.youtube.com/@gitbutlerapp"),
-            "help/x" => open::that("https://x.com/gitbutler"),
+            "help/discord" => open::open_url("https://discord.com/invite/MmFkmaJ42D"),
+            "help/youtube" => open::open_url("https://www.youtube.com/@gitbutlerapp"),
+            "help/x" => open::open_url("https://x.com/gitbutler"),
```

Comment:
Usually I avoid duplicating parts of the name, so `open::url()` would probably be all it needs here.

Reply:
Hmm so I'd like to make it `open::url`, that does look and feel nice. However then the fn is also just called `open` from the point of view of `invoke` on the Svelte side which feels a bit too abstract - open what? - ya know. 

So I just imported the fn directly and referred to it as `open_url` instead of `open::open_url`. What do you think?

Reply:
Now it's definitely looking better, and clearly, it's a nit, and often it comes down to the question if everything should be uniquely named if imported directly, or if things should be referred to by module and thus each piece of the path can be short.

Gitoxide definitely does the latter, whereas `git2` does the former, and I don't think I have an answer as to what's *always* the best.